### PR TITLE
feat(login): replace native inputs and button with chakra ui lib, in …

### PR DESCRIPTION
**Pull request by pablolizardo**
…order to improve ui

re #87

### CODE REVIEW

**Review by santiagolizardo — COMMENTED** [review]
Changes look good, although I can't approve because the loading animation is not showing properly. (the spinning wheel shows some weird lines)
Do you see the same?

File: src/components/auth/Login.js [hunk 1]
```diff
@@ -45,18 +45,38 @@ const Login = () => {
...
-                            <fieldset>
-                                <legend>Login</legend>
-                                <label htmlFor="inputUsername" className="sr-only">Username</label>
-                                <input type="text" id="inputUsername" onChange={handleUsername}
+                            <FormControl as="fieldset">
+                                <Heading as="legend" size="lg" mb="3" > Login</Heading>
+                                <Input
+                                    autoFocus
+                                    isRequired
+                                    type={"text"}
```
Comment (santiagolizardo) [inline src/components/auth/Login.js:53]: When there is nothing special/dynamic about an attribute better to use the standard form:
```suggestion
                                    type="text"
```

File: src/components/auth/Login.js [hunk 2]
```diff
@@ -45,18 +45,38 @@ const Login = () => {
...
+                                />
                                 <label htmlFor="inputPassword" className="sr-only">Password</label>
-                                <input type="password" id="inputPassword" onChange={handlePassword}
-                                    placeholder="Password" required />
-                                <PrimaryButton
-                                    type="submit">{!loading ? "Sign in" : "Processing..."}</PrimaryButton>
-                            </fieldset>
+                                <Input
+                                    isRequired
+                                    type={"password"}
```
Comment (santiagolizardo) [inline src/components/auth/Login.js:61]: When there is nothing special/dynamic about an attribute better to use the standard form:
```suggestion
                                    type="password"
```

File: src/components/auth/Login.js (hunk 1 above)
Comment (pablolizardo) [reply src/components/auth/Login.js]: done!

**Comment by pablolizardo** [comment]
> Changes look good, although I can't approve because the loading animation is not showing properly. (the spinning wheel shows some weird lines)
> Do you see the same?

yes! my mistake, it was a conflict with the scss file. now it's fixed

**Review by santiagolizardo — APPROVED** [review]

**Pull request merged**